README.md

```diff
@@ -64,7 +64,24 @@ To get the source code from GitHub, type
 1. Make sure you have installed **[Compoeser](https://getcomposer.org/doc/00-intro.md#introduction)** in your system.
 It can be installed locally or globally depending on your own needs.
 
-2. Execute the following command in order to get this package:
+2. You can edit your `composer.json` file manually or directly add the requirement via CLI:
+
+###Manually edit
+Create or edit your **composer.json** in the root of your project and add the following code to it:
```

Comment:
For this section, I would just use the original text, so new users will not be distracted by composer operations.